pkg/metrics/collector/conntrack.go

```diff
@@ -81,13 +97,26 @@ func parseHeader(line string) (*conntrackIndices, error) {
 	indices.numFields = len(nameParts)
 	for i, v := range nameParts {
 		switch v {
+		case "found":
+			indices.found = i
+		case "invalid":
+			indices.invalid = i
+		case "insert":
+			indices.drop = i
 		case "insert_failed":
 			indices.insertFailed = i
 		case "drop":
 			indices.drop = i
+		case "early_drop":
+			indices.earlyDrop = i
+		case "search_researt":
+			indices.searchRestart = i
 		}
 	}
-	if indices.insertFailed == -1 || indices.drop == -1 {
+	if indices.found == -1 || indices.invalid == -1 ||
```

Comment:
Looks like this won't work as we didn't explicitly assign the other fields `-1` initially?

Reply:
Done. Set all the indices to `-1` before this judgment.